styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,15 @@
-.footer{
-  
-}
\ No newline at end of file
+.rug{
+  .text37{
+    font-size: 37px;
+    line-height: 24px;
+  }
+  &, *{
```

Comment:
I can add class="white" to each of these elements but since everything in the rug will always be white, this seems a lot less hefty.